ToolsForHigherHomologicalAlgebra/PackageInfo.g

```diff
@@ -74,6 +74,7 @@ Dependencies := rec(
         [
           [ "AutoDoc", ">=2018.02.14" ],
           [ "CAP", ">=2021.11-05" ],
+          [ "FunctorCategories", ">= 2021.11-06" ],
```

Comment:
Thank you. Everything looks fine except this dependency. The package indeed contains a code to handle `FunctorCategories`, however, it is only loaded if one calls `EnahncePackage( "FunctorCategories" );`